src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/BerConverter.cs

```diff
@@ -191,8 +207,31 @@ public static byte[] Encode(string format, params object[] value)
 
                     byte[][] tempValue = (byte[][])value[valueCount];
 
-                    error = EncodingMultiByteArrayHelper(berElement, tempValue, fmt);
+                    error = EncodingMultiByteArrayHelper(berElement, tempValue, fmt, tag);
+
+                    valueCount++;
+                }
+                else if (fmt == 't')
```

Comment:
I'm sure I'm missing something obvious but I'm not understanding why we mvoed the t here. even when unsupported seems to me we are doing sort of the same than we were doing before right?